### PR TITLE
Dbms docs sec 5 6 7 added

### PR DESCRIPTION
**Pull request by Vipullakum007**
## Related Issue

Fixed : #735 

## Description

i have added following section under DBMS tutorials
image

using 
1 . mermaid diagram
image

image

2. add table according to topic more readable to user
image

3. comparision/Difference with using table for more readable
image

4. algebra like this

image

5. note also
image

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

Screenshot (726)
Screenshot (727)
Screenshot (728)
Screenshot (729)
Screenshot (730)
Screenshot (731)

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Comment by Vipullakum007** [comment]
hey @Ajay-Dhangar , **_please make it level 3_** 

i have added following section under DBMS tutorials
image

using 
1 . mermaid diagram
image

image

2. add table according to topic more readable to user
image

3. comparision/Difference with using table for more readable
image

4. algebra like this

image

5. note also
image

File: docs/DBMS/Indexing And Hashing/indexing.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+1. **Insertion at Leaf Node:**
+    - If a leaf node overflows, split node into two parts.
+    - Partition at i = ⌊(m+1)/2⌋.
+    - First i entries are stored in one node.
+    - Rest of the entries (i+1 onwards) are moved to a new node.
+    - ith key is duplicated at the parent of the leaf.
+
+2. **Insertion at Non-leaf Node:**
+    - Split node into two parts.
+    - Partition the node at i = ⌈(m+1)/2⌉.
```
Comment (Ajay-Dhangar) [inline docs/DBMS/Indexing And Hashing/indexing.md:120]: replace `i = ⌊(m+1)/2⌋` to `$i = ⌊(m+1)/2⌋$`

File: docs/DBMS/Storage And File Structure/dbms-file-structure.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    B --> E[Delete]
+    B --> F[Update]
+    C --> G[Select]
+    C --> H[Filter]
+```
+
+The organization of data inside a file plays a major role in how efficiently these operations can be performed. The method used to locate the file pointer to a desired record inside a file varies based on whether the records are arranged sequentially or clustered.
+
+In summary, understanding the various file structures and their operations is crucial for efficient database management, ensuring optimal performance and reliability.
+```
```
Comment (Ajay-Dhangar) [inline docs/DBMS/Storage And File Structure/dbms-file-structure.md:96]: remove <code>```</code>

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
now update basic code based on suggestions for good practice

**Comment by Vipullakum007** [comment]
hey @Ajay-Dhangar  ,  updated as per your suggestion you can review it.

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**